zeg/collections.py

```diff
@@ -47,6 +47,10 @@ def create(log, session, args):
         if key in configuration:
             coll[key] = configuration[key]
 
+    # replace empty description with an empty string
+    if coll["description"] is None:
```

Comment:
If `description` isn't in coll at all (which is valid config) this will break. Instead, I think we want:

```suggestion
    if 'description' in coll and coll["description"] is None:
```